library/parse_backup_metadata.py

```diff
@@ -18,7 +18,7 @@
 #          - Full path (including file name) to metadata.yml
 #        required: true
 
-SUPPORTED_VERSIONS = ["6.2", "6.3", "6.4", "6.5", "6.6", "6.7"]
+SUPPORTED_VERSIONS = ["6.2", "6.3", "6.4", "6.5", "6.6", "6.7", "6.8"]
```

Comment:
same here

Reply:
@evgeni I think we should create a new bug for removing the support below 6.5.

Since this request is about adding support to 6.8, I did not remove those files.

Reply:
Right, we can, but then we need to put `get_puppet_version` back :)